RELEASES.md

```diff
@@ -1,3 +1,22 @@
+<a name="0.20.1"></a>
+## 0.20.1 (2023-10-**)
+
+#### Features
+
+* Dangling symlinks in crate no longer cause build failure (thanks to legeana for the report!)
+* Lalrpop now uses the ascii-aware space regex when the unicode feature is not enabled (thanks to QuarticCat!)
+* Lalrpop no longer depends on the `is-terminal` crate (thanks to Kmeakin!)
+* Better performance with the default lexer using the underlying `regex-automata` crate (thanks to QuarticCat!)
+* Allow the catch-all `_` case for token matching can now be set to a higher precedence in match (thanks to fpoli!)
+* Fewer clippy lints triggered in generated code
+
+#### Compatibility note
+
+* MSRV increased to `1.70` in accordance to the MSRV policy.
```

Comment:
The phrase "in accordance to the MSRV policy" makes it sound like the increase is due to some policy rather than the is_terminal dependency requiring it.  Maybe just end at "MSRV increased to 1.70"?